bin/start_kube.sh

```diff
@@ -254,7 +262,8 @@ fi
 if pulumi config get prometheus:adminpass -C ${script_dir}/../pulumi/python/config >/dev/null 2>&1; then
   echo "Password found for grafana admin account"
 else
-  echo "Create a password for the grafana admin user"
+  echo "Create a password for the grafana admin user; this password will be used to access the Grafana dashboard"
```

Comment:
Two comments:
1. When I initially went through the setup process, I was alarmed that this password showed up in plaintext mainly just from my gut experience but because it is a big deal.  Although it's just a test app, I wonder if it would be better optics wise for us to have it stay hidden but advise the user that they will need to keep track of the password.
2. If the user needs the password to access some UI, perhaps this would be a good place to print out the url where the password can be applied.  Maybe not here, but at the end?